data/pigui/views/mainmenu.lua

```diff
@@ -271,6 +270,6 @@ local function showMainMenu()
 	callModules('modal')
 end -- showMainMenu
 
-ui.registerHandler('mainMenu',function(delta)
+ui.registerHandler('mainMenu',function(_)
```

Comment:
If you're going to change this, you can just write it as `ui.registerHandler('mainMenu', showMainMenu)`.